Makefile

```diff
@@ -22,7 +22,13 @@ install:
 # TODO: build this manually instead of requiring buildx
 test/out/img.tar: test/image/Dockerfile test/image/src/main.rs test/image/Cargo.toml test/image/Cargo.lock
 	mkdir -p $(@D)
+	docker buildx rm wasmbuilder || true
```

Comment:
This is going to mess up my build cache.
Maybe we can instruct people to set a different builder instead?

Note that you should be able to pass one in by setting $BUILDX_BUILDER already.

Reply:
only for this builder? This only gets run once do the file marker `test/out/img.tar`

I didn't know about $BUILDX_BUILDER.  I tried it and it worked. Will be easy enough to just give that hint in the readme.

Reply:
I wanted to get away from requiring docker for building this.
I'll work on that.

Reply:
sounds good.  Closing in favor of https://github.com/containerd/runwasi/pull/41